tools/block-generator/README.md

```diff
@@ -92,24 +94,59 @@ final_overall_transactions_per_second:8493.40
 final_uptime_seconds:3600.06
 ```
 
-Here is the help output:
+Here is the help output for **runner**:
 ```bash
 ~$ ./block-generator runner -h
-Run test suite and collect results.
+Run an automated test suite using the block-generator daemon and a provided conduit binary. Results are captured to a specified output directory.
 
 Usage:
   block-generator runner [flags]
 
 Flags:
-      --cpuprofile string                   Path where Indexer writes its CPU profile.
+  -i, --conduit-binary string               Path to conduit binary.
+      --cpuprofile string                   Path where conduit writes its CPU profile.
   -h, --help                                help for runner
-  -i, --indexer-binary string               Path to indexer binary.
-  -p, --indexer-port uint                   Port to start the server at. This is useful if you have a prometheus server for collecting additional data. (default 4010)
-  -l, --log-level string                    LogLevel to use when starting Indexer. [error, warn, info, debug, trace] (default "error")
+  -k, --keep-data-dir                       If set the validator will not delete the data directory after tests complete.
```

Comment:
Is this a new flag (or perhaps an inherited one)? I didn't see it in the `tools/block-generator/run_tests.sh` help command

Reply:
`run_tests.sh` currently is only passing thru [a few of the flags](https://github.com/algorand/go-algorand/blob/766afeec44d40eceaff37135031b0cc02caaca9f/tools/block-generator/run_tests.sh#L89).

I'm not familiar enough with the tests runner to have a strong opinion on whether we should pass more thru. @shiqizng @winder any thoughts?

Reply:
> run_tests.sh currently is only passing thru [a few of the flags](https://github.com/algorand/go-algorand/blob/766afeec44d40eceaff37135031b0cc02caaca9f/tools/block-generator/run_tests.sh#L89).

Oh is this separate from the block generator runner? I was just wondering why there was a diff (addition) in the README.

Reply:
I just updated the README by re-running the tool incorporating Shiqi's and my changes, and then pasted in the new outputs.

Reply:
Gotcha, it's clearer to me now - I reproduced this so I will resolve this thread, thanks!